src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubRepositoryDetail.java

```diff
@@ -0,0 +1,59 @@
+package org.jenkinsci.plugins.github_branch_source;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+import hudson.model.Actionable;
+import hudson.model.Run;
+import jenkins.model.details.Detail;
+import jenkins.model.details.DetailGroup;
+import jenkins.scm.api.SCMDetailGroup;
+import jenkins.scm.api.SCMSource;
+
+public class GitHubRepositoryDetail extends Detail {
```

Comment:
I think it would be good to get some test coverage of these.

Either mocking or wiremock should allow it.